packages/database/src/TupaiaDatabase.js

```diff
@@ -572,6 +593,7 @@ function addWhereClause(connection, baseQuery, where) {
       throw new Error(`Cannot compare using ${comparisonType}`);
     }
 
+    // TODO: Replace with knex json where functions, eg. whereJsonPath
```

Comment:
Saw these when checking out the latest knex functions. Didn't quite wanna touch it here since this code has been a touch flaky in the past...